website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step12-add-nodepool.sh

```diff
@@ -23,14 +23,15 @@ spec:
           operator: Gt
           values: ["2"]
       nodeClassRef:
-        apiVersion: karpenter.k8s.aws/v1
+        group: karpenter.k8s.aws
```

Comment:
Would it be possible to make this update for v1.0 and preview folders as well?

Reply:
@engedaam 
I fixed v1.0 and preview.